src/adapter.js

```diff
@@ -12,7 +12,51 @@ export const mediaTypes = [
 ];
 
 export function detect(source) {
-  return !!source.match(/"?swagger"?:\s*["']2\.0["']/g);
+  return !!(_.isString(source)
+    ? source.match(/"?swagger"?:\s*["']2\.0["']/g)
+    : source.swagger === '2.0');
+}
+
+// Test whether a key is a special Swagger extension.
+function isExtension(value, key) {
+  return key.indexOf('x-') === 0;
+}
+
+// Test whether tags can be treated as resource groups, and if so it sets a
+// group name for each resource (used later to create groups).
```

Comment:
This feature is important for documentation and organization. All of the real-world examples I have seen use tags in such a way that we can treat them as resource groups. Since refract doesn't support a tagging feature and if/when it gets added it means an extra burden on tooling authors (e.g. Aglio then needs to support **both** resource groups *and* tags) I'm for trying it this way.

If we discover some issues with it then I am okay reevaluating this position, but at the moment (with no user feedback) I believe that users will see this feature as treating Swagger more like a first class citizen in Fury.